src/power_grid_model/enum.py

```diff
@@ -67,6 +67,27 @@ class CalculationMethod(IntEnum):
     iec60909 = 5
 
 
```

Comment:
TBD: should we add an enum `ControlSide` here?

```suggestion
class ControlSide(IntEnum):
    """Control Sides for Regulators on Branch Objects"""

    from_side = 0
    to_side = 1
    side_1 = 0
    side_2 = 1
    side_3 = 2


```

* [ ] If so: should we add a validation check to check that the from_side/to_side/side_1/side_2/side_3 is in range of the transformer type
* [ ] If not: should we add a validation check to check
  * [ ] that the correct enum type is being used and/or
  * [ ] that it is in range of the transformer type

Reply:
I dont see an extra enum being useful. The intention is sufficiently conveyed by using BranchSide or Branch3Side itself.
A data validation check would be nice.

Reply:
Created a JIRA ticket for this